ptr_tests.py

```diff
@@ -57,7 +56,7 @@ def return_specific_pid(*args: Any, **kwargs: Any) -> int:
 
 
 def return_specific_tmp(*args: Any, **kwargs: Any) -> str:
-    return "/tmp"
+    return "C:\\tmp" if ptr.WINDOWS else "/tmp"
```

Comment:
`tempfile.gettempdir` is your friend. I see you're using it down below, why not here?

Reply:
I was trying to be more explicit but I guess it should always match on respective platforms.